cmake/libusrsctp.cmake

```diff
@@ -31,6 +31,11 @@ elseif (APPLE)
         -UINET
         -UINET6
     )
+elseif (CMAKE_SYSTEM_NAME STREQUAL "OpenBSD")
```

Comment:
Maybe it's better to use external usrsctp with its upstream build scripts for you?

Reply:
`tg_owt` development on OpenBSD (for me) started without an existing `usrsctp` port, so these bits were and still are useful to me.

Now `usersctp` is built as a standalone port providing a shared library and I will most certainly use that when building `tg_owt`.

Still, I see no downside in supporting both ways, hence this patch.

Reply:
In order to use an external `usersctp` library, one currently has to remove the following line anyway:
https://github.com/desktop-app/tg_owt/blob/429a6869e4a164e0aad2d8657db341d56f9a6a6f/CMakeLists.txt#L71
Which is what FreeBSD's official port is doing already:
https://cgit.freebsd.org/ports/tree/net-im/tg_owt/files/patch-CMakeLists.txt?id=11338fde41532258b8a30e80c1c46a999240dd9b#n15

I plan to add a proper flag for this in the future, but that is stuff for another diff  and `usrsctp` is not the only dependency behaving like this.

Reply:
You can't move this line under a condition, snap build starts to fail otherwise (see #66)

Reply:
Then I'll stick with simple removal of those lines (until a better solution is found).

Reply:
ok

Reply:
Linking https://github.com/desktop-app/tg_owt/pull/85 here as my current solution to dependency handling.